packages/cli/package.json

```diff
@@ -65,38 +65,34 @@
   },
   "config": {
     "templateDependencies": {
-      "//1": "This file contains dependency/version used by project templates",
-      "//2": "It can be updated by `npm run update-template-deps` under `loopback-next`",
```

Comment:
@raymondfeng was it your intention to remove these comments? I don't mind, just asking.